frontend/src/App.js

```diff
@@ -1,25 +1,39 @@
-import logo from './logo.svg';
-import './App.css';
+import {Route, Switch, BrowserRouter as Router} from 'react-router-dom';
+import AppHeader from "./components/AppHeader";
+import SearchProducts from "./components/SearchProducts";
+import {useEffect, useState} from "react";
+import {getUserProductsList} from "./service/ApiService";
+import ProductList from "./components/ProductList";
+
+
+
 
 function App() {
+    const [products, setProducts] = useState([])
+
+    useEffect(() => {
+        getUserProductsList()
+            .then(setProducts)
+            .catch((error) => console.error(error))
+    }, [])
```

Comment:
if you do not need this information in all of your app components included in `App.js` you may want to consider moving it to your e.g. `<SearchProducts>` component or using context to keep your `App.js` minimal and easily readable